changes/probe.py

```diff
@@ -58,11 +68,20 @@ def has_changelog():
 
 
 def has_readme():
-    """README.md"""
+    """README"""
+    readme = None
+    message = 'Create a README file'
+    for filename in os.listdir("."):
+        readme = re.match(r"README($|\.\w{2,})", filename, re.I)
```

Comment:
I thought this was safer because it will only match README (the word) and then with or without a file extension. If I had used a glob wildcard (and then split off the extension) that would match words starting with README like READMEABEDTIMESTORYMOM. If I look at it now, I guess you could first split between filename and file extension and then do the check, but otherwise I also like that you can retrieve the filename through the capture group here. Do you have any concerns about using regex?